### DISCUSSION TITLE
Vite Migration - function is not exported by app/my.server.js

### DISCUSSION THREAD

**Discussion by BryceBlankinship** [remix-run/remix#7987] [Q&A] [answered]
I've been moving through the unstable vite migration docs, and encountered this issue when running `vite build`: `"getAllProducts" is not exported by "app/sku.server.js", imported by "app/routes/products/$product.jsx".`

Any help on this would be appreciated, the import is from the correct location and the function is exported properly. If I comment out this line, vite just moves to the next instance of a server function import. Maybe I have to do something with vite config on all my server files?

**Comment by hi-ogawa** [comment, marked as answer]
What you seeing might be a known issue tracked here https://github.com/remix-run/remix/issues/7924
A few workaround is suggested there as well.
